pages/common/read.md

```diff
@@ -19,7 +19,7 @@
 
 `read -d {{new_delimiter}} {{variable}}`
 
-- Do not let backslash (\) act as an escape character:
+- Do not let backslash (\\) act as an escape character:
```

Comment:
```suggestion
- Do not let backslash act as an escape character:
```

Reply:
If other maintainers prefer this, happy to run with it. I personally have many friends that mix up forward slash (/) and backslash (\\), so I think having the character helps.

Reply:
> If other maintainers prefer this, happy to run with it. I personally have many friends that mix up forward slash (`/`) and backslash (`\`), so I think having the character helps.

I definitely agree that it might be confusing to some. So, I think we can go with this change for proper rendering.